docs/api/plugins/aea_ledger_ethereum/ethereum.md

```diff
@@ -898,12 +898,28 @@ Get all transfer events derived from a transaction.
 
 - `contract_instance`: the contract
 - `tx_hash`: the transaction hash
-- `target_address`: optional address to filter tranfer events to just those that affect it
+- `target_address`: optional address to filter transfer events to just those that affect it
 
 **Returns**:
 
 the transfer logs
 
+<a id="plugins.aea-ledger-ethereum.aea_ledger_ethereum.ethereum.EthereumApi.bundle_and_send"></a>
+
+#### bundle`_`and`_`send
+
+```python
+def bundle_and_send(raw_signed_transactions: List[str],
+                    target_blocks: List[int]) -> Optional[List[str]]
+```
+
+Simulate and send a bundle of transactions.
+
```

Comment:
Same here

Reply:
Done in 94b96880e7a65eba064b81727620a07a530d5957.